.gitpod.Dockerfile

```diff
@@ -0,0 +1,52 @@
+FROM eu.gcr.io/gitpod-core-dev/dev/dev-environment:cw-bump-leeway.29
+
+# Install Xvfb, JavaFX-helpers, Openbox window manager and emacs GTK
+RUN sudo install-packages xvfb x11vnc xterm openjfx libopenjfx-java openbox \
+    emacs-gtk
+
+# Install Cypress dependencies.
+RUN sudo apt-get update \
+    && sudo DEBIAN_FRONTEND=noninteractive apt-get install -y \
+    libgtk2.0-0 \
+    libgtk-3-0 \
+    libgbm-dev \
+    libnotify-dev \
+    libgconf-2-4 \
+    libnss3 \
+    libxss1 \
+    libasound2 \
+    libxtst6 \
+    xauth \
+    xvfb \
+    && sudo rm -rf /var/lib/apt/lists/*
+
+# Overwrite this env variable to use a different window manager
```

Comment:
E.g, [this](https://github.com/gitpod-io/gitpod/blob/main/dev/image/Dockerfile#L5) should be `gitpod/workspace-full-vnc:latest`. Then all scripts can be removed and their paths adapted.

Reply:
🤞 Ok, I think I fixed the docker conf in 5619a174299759e47904f14969ca0a61e70c7049 but I don't know how to launch a new build for the image so that a new release appears [on GCR](https://console.cloud.google.com/gcr/images/gitpod-core-dev/eu/dev/dev-environment) and I can then replace the original `:cw-bump-leeway.29` 🐳  and have a working environment again.

Any pointers? 😊

I need this to work in order to update the test that will fail after the rebase from `main` as [mentioned in a previous comment](https://github.com/gitpod-io/gitpod/pull/7611#issuecomment-1014470441).

Reply:
Whoops, answer went here: https://github.com/gitpod-io/gitpod/pull/7611#issuecomment-1016327106

Reply:
Thanks, @geropl, it worked!

A couple of problems though:

- all the files where the image is specified are now conflicting
- I found out this hacky bit from the `workspace-full-vnc` image:
```sh
# This is a bit of a hack. At the moment we have no means of starting background
# tasks from a Dockerfile. This workaround checks, on each bashrc eval, if the X
# server is running on screen 0, and if not starts Xvfb, x11vnc and novnc.
RUN echo "export DISPLAY=:0" >> ~/.bashrc
RUN echo "[ ! -e /tmp/.X0-lock ] && (/usr/bin/start-vnc-session.sh &> /tmp/display-\${DISPLAY}.log)" >> ~/.bashrc
```
makes Cypress work only if you are using bash, or you load bash then switch back to your favorite shell. For example my default shell is zsh.
I'll add a note in the README for now, but it feels like something that should be addressed sooner or later.

Reply:
> all the files where the image is specified are now conflicting

This is a general problem with how we manage this dev image.  Conflicts are just very likely to happen. :grimacing: 

@trumbitta I propose the following: you sort out the other todos, incl. rebase. Then ping me for another build, and we put the resulting image on top and merge quickly. Does that sound good?

> makes Cypress work only if you are using bash, or you load bash then switch back to your favorite shell. For example my default shell is zsh.

Agree this is hacky. I'd leave it for now, and tackle that in a separate PR. Not sure, maybe `/etc/profile` is a better place for this.

Reply:
> @trumbitta I propose the following: you sort out the other todos, incl. rebase. Then ping me for another build, and we put the resulting image on top and merge quickly. Does that sound good?
> 

🤝 perfect

Reply:
@geropl I think we are ready for the new build!